docs/source/resources.rst

```diff
@@ -30,6 +30,24 @@ Here is how you can register a resource::
             _USERS[len(_USERS) + 1] = self.request.json_body
             return True
 
+Here a example of imperative way defining a resource::
+
+    from cornice.resource import add_resource, add_view
```

Comment:
I would rather import `resource` and use `resource.add_view` and `resource.add_resource`.